scripts/ci/jobs/gke_version_compatibility_tests.py

```diff
@@ -5,41 +5,45 @@
 """
 import logging
 import os
-import sys
 import subprocess
+import sys
 from clusters import GKECluster
+from collections import namedtuple
 from compatibility_test import make_compatibility_test_runner
 from get_latest_helm_chart_versions import get_latest_helm_chart_versions
 
+# start logging
+logging.basicConfig(stream=sys.stdout, level=logging.DEBUG)
 
 # set required test parameters
 os.environ["ORCHESTRATOR_FLAVOR"] = "k8s"
 os.environ["ROX_POSTGRES_DATASTORE"] = "true"
 
 central_chart_versions = get_latest_helm_chart_versions("stackrox-central-services", 2)
 sensor_chart_versions = get_latest_helm_chart_versions("stackrox-secured-cluster-services", 3)
-latest_tag = subprocess.check_output("make tag", shell=True).strip().decode("utf-8")
+makefile_path = os.path.abspath(os.path.join(os.path.dirname( __file__ ), '../../..'))
```

Comment:
[minor] Pathlib is much more modern alternative to `os.path` functions. Pathlib also works (mostly) everywhere string paths are expected.
With pathlib, this line would become `makefile_path = pathlib.Path(__file__).parent.parent.parent.parent` or `makefile_path = pathlib.Path(__file__).parent / '../../..'`

Reply:
Sure, done